index.html

```diff
@@ -5132,6 +5132,12 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type MUST NOT be used with <dfn>embedded proof</dfn>.
```

Comment:
```suggestion
    This media type MUST NOT be used with <dfn>embedded proof</dfn>.
```

The presumption here is that we're going to have a separate media type for a verifiable credential, right? That is, this PR goes in and we must create a new media type for credentials that use embedded proofs, such as `application/verifiable-credential+ld+json`?

Regarding testability of this statement... given that this is a hard normative requirement, how are we testing it?

Reply:
Why is this a useful requirement?

I also wonder what kind of trouble we might create with proof sets -- credentials having multiple proofs or proofs that aren't about securing the credential but are proofs, for example, that the credential data was an input to a captcha / PoW / timestamp-generator / whatever.

Reply:
> The presumption here, with which I think I agree, is that we're going to have a separate media type for a verifiable credential, right? 

Yes, this proposal attempts to close the door to an alternative where we refer to both "Credential" and "Verifiable Credential" as `application/credential+ld+json`... I believe that would be a mistake, in a similar spirit to how `alg:none` was a mistake.

Reply:
@dlongley I agree, related issues will impact `proof sets` and `proof chains`:

- https://w3c.github.io/vc-data-integrity/#proof-sets

Specifically, if you wanted to "add another proof", you would be forced to use `application/verifiable-credential+ld+json` and not `application/credential+ld+json` for the "inbound" media type.

For some hypothetical APIs, that seems like a safety feature though, considering that you cannot "add to a proof set" or "add to a proof chain" if no proof exists in the underlying object.

However, if you wanted to make a `proof set` API that took `application/credential+ld+json` and produced an `application/verifiable-credential+ld+json` based on a `proof chain`... you could use this media type in the API.

Reply:
If I receive an `application/verifiable-credential+ld+json` document and don't understand that media type, but *do* understand `application/ld+json` or `application/json`, can I get anything out of that document?  If not (which I believe is the intention), then `application/verifiable-credential+ld+json` is not an appropriately constructed media type.

Reply:
> can I get anything out of that document?

Not sure what this means.

`+ld+json` intention of this suffix is to communicate that the document is JSON-LD, which can also be treated as JSON.

The current spec for `application/credential+ld+json` says this:

> Resources that use the "application/credential+ld+json" Media Type are required to conform to all of the requirements for the "application/ld+json" Media Type and are therefore subject to the same encoding considerations specified in Section 11 of [[RFC7159](https://w3c.github.io/vc-data-model/#bib-rfc7159)].

I assume you are asking if the registration for `application/verifiable-credential+ld+json` would be similar... I assume it would.

Reply:
> [@OR13] The current spec for `application/credential+ld+json` says this:
>
> Resources that use the "application/credential+ld+json" Media Type are required to conform to all of the requirements for the "application/ld+json" Media Type and are therefore subject to the same encoding considerations specified in Section 11 of [[RFC7159](https://w3c.github.io/vc-data-model/#bib-rfc7159)].

Ah! I missed that definition. OK, that's good.

The next question is about the definitions for these two media types:
* **`application/verifiable-credential+ld+json`** (with hyphenated `verifiable`**`-`**`credential`) — which *could* be a subset, *or* a disjoint set, *or* a partially intersecting set of `application/credential+ld+json`, by the rules of media types
* **`application/verifiable+credential+ld+json`** (with additive `verifiable`**`+`**`credential`) — which *must* be a subset of `application/credential+ld+json`, by the rules of media types

Relatedly -- 
* Are Verifiable Credentials Signed _**but not** Encrypted_? Signed _**and** Encrypted_? If the latter, _wholly_ or _partially_ encrypted (i.e., are some attributes and/or values unencrypted)? I have been thinking Signed and Wholly Encrypted, but it seems I may have misunderstood something along the way.

Reply:
I'm curious what an _and encrypted_ media type would look like.

Reply:
Sorry I missed this:

> Are Verifiable Credentials Signed but not Encrypted? Signed and Encrypted? If the latter, wholly or partially encrypted (i.e., are some attributes and/or values unencrypted)?

In version 1.1 there is no support for encrypted credentials or presentations.

I believe encryption should be considered, especially for presentations, but that is probably a topic for other issues: 

- https://github.com/w3c/vc-data-model/issues/938
- https://github.com/w3c/vc-data-model/issues/971

Reply:
Resolving this thread based on the merged suggestion here: https://github.com/w3c/vc-data-model/pull/1014/commits/a99fe01b917e5ce160cf78031e2d6e12546d6630